src/main/java/io/buildpacks/example/sample/CalculatorService.java

```diff
@@ -23,4 +23,35 @@ public int divide(int a, int b) throws IllegalArgumentException {
         }
```

Comment:
1. **Misleading Method Names**:
    - The method `public int sum(int a, int b)` has a misleading name, as it performs addition, not sum. Consider renaming it to `add` for clarity.